lib/util/util.CombiningMessageProvider.js

```diff
@@ -0,0 +1,53 @@
+this.util = this.util || {};
+
+util.CombiningMessageProvider = ( function() {
+	'use strict';
+
+	/**
+	 * Providing messages using two message providers
+	 * @class util.CombiningMessageProvider
+	 * @licence GNU GPL v2+
+	 * @author H. Snater < mediawiki@snater.com >
+	 * @author Adrian Heine < adrian.heine@wikimedia.de >
+	 *
+	 * @constructor
+	 *
+	 * @param {util.MessageProvider} a
+	 * @param {util.MessageProvider} b
+	 */
+	function CombiningMessageProvider( a, b ) {
+		this._a = a;
+		this._b = b;
+	}
+
+	CombiningMessageProvider.prototype = {
+		constructor: CombiningMessageProvider,
+
+		/**
+		 * @property {util.MessageProvider} _a
```

Comment:
Note: No need to repeat the member name if not defaulting to a particular value.